### PR TITLE
Add ability to toggle fullscreen

### PR DESCRIPTION
**Pull request by GregKWhite**
Pretty self explanatory. Allows users to resize streams to be fullscreen.

### Regular
screen shot 2017-05-28 at 4 42 40 pm

### Fullscreen
screen shot 2017-05-28 at 4 42 48 pm

### CODE REVIEW

**Review by maxcell — APPROVED** [review]
👍 LGTM

File: src/styles/playback/stream-player.styl [hunk 1]
```diff
@@ -26,6 +26,11 @@
     .playback-controls
       opacity 1
 
+  &:-webkit-full-screen {
+    width 100%
+    height 100%
+  }
```
Comment (stevenpetryk) [inline src/styles/playback/stream-player.styl:32]: Nice! Didn't even know this was a thing.

File: src/components/playback/StreamPlayer.js [hunk 2]
```diff
@@ -81,20 +82,55 @@ export default class StreamPlayer extends React.Component {
...
+  requestFullscreen () {
+    if (this.refs.streamPlayer.requestFullscreen) {
+      this.refs.streamPlayer.requestFullscreen()
+    } else if (this.refs.streamPlayer.mozRequestFullScreen) {
+      this.refs.streamPlayer.mozRequestFullScreen()
+    } else if (this.refs.streamPlayer.webkitRequestFullscreen) {
+      this.refs.streamPlayer.webkitRequestFullscreen()
+    } else if (this.refs.streamPlayer.msRequestFullscreen) {
+      this.refs.streamPlayer.msRequestFullscreen()
+    }
```
Comment (stevenpetryk) [inline src/components/playback/StreamPlayer.js]: Possibly cleaner way to do this (for `exitFullscreen` as well):

```js
(
  this.refs.streamPlayer.requestFullscreen ||
  this.refs.streamPlayer.mozRequestFullscreen ||
  this.refs.streamPlayer.webkitRequestFullscreen ||
  this.refs.streamPlayer.msRequestFullscreen ||
  () => {}
)()
```

**Review by stevenpetryk — APPROVED** [review]
Again, looks great. Just style things.

File: src/components/playback/StreamPlayer.js (hunk 2 above)
Comment (GregKWhite) [reply src/components/playback/StreamPlayer.js]: Good call! I had to make some minor tweaks to get it to work, but that's so much cleaner. Thanks!

**Pull request merged**